package.json

```
@@ -20,7 +23,8 @@
     "test:analyze": "npm run build:production -- --env.analyze=true --json --progress --profile > /tmp/stats.json && cp ./dist/assets/js/*.js /tmp && webpack-bundle-analyzer /tmp/stats.json",
     "test:js": "wp-scripts test-unit-js --config=tests/js/jest.config.js",
     "test:js:watch": "wp-scripts test-unit-js --config=tests/js/jest.config.js --watch",
-    "test:e2e": "WP_BASE_URL=http://localhost:9002 wp-scripts test-e2e --config=tests/e2e/jest.config.js",
+    "pretest:e2e": "./bin/local-env/env-check.sh",
+    "test:e2e": "WP_BASE_URL=$npm_package_config_E2E_URL wp-scripts test-e2e --config=tests/e2e/jest.config.js",
```

Let's pass the URL using the command option instead of the env variable. It's more clear what the actual command is this way I think:

```suggestion
    "test:e2e": "wp-scripts test-e2e --config=tests/e2e/jest.config.js --wordpress-base-url=$npm_package_config_E2E_URL",
```